eva_cttv_pipeline/clinvar_xml_io/tests/test_clinvar_dataset.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from eva_cttv_pipeline.clinvar_xml_io.clinvar_xml_io import ClinVarDataset
+
+
+resources_dir = os.path.join(os.path.dirname(__file__), 'resources')
+
+
+def test_dataset_write():
+    input_file = os.path.join(resources_dir, 'test_clinvar_dataset.xml.gz')
+    output_file = os.path.join(resources_dir, 'test_output.xml.gz')
+
+    input_dataset = ClinVarDataset(input_file)
+    input_dataset.write(output_file)
```

Comment:
I assume ``test_output.xml.gz`` is the annotated version of ClinVar's that we would also distribute some how in case people want to use it, right?

Reply:
Or are we missing the annotation chunk in between?

Reply:
Yes that's right, there's no annotation yet and we're just outputting the [raw RCV XML](https://github.com/EBIvariation/eva-opentargets/blob/4b10260c745e67649bd791579c8e4753ec155929/eva_cttv_pipeline/clinvar_xml_io/clinvar_xml_io/clinvar_record.py#L59) for each record.  The annotation will be coming in a subsequent PR.